content_scripts/vimium_frontend.coffee

```diff
@@ -274,23 +274,22 @@ setScrollPosition = ({ scrollX, scrollY }) ->
 #
 # Called from the backend in order to change frame focus.
 #
-window.focusThisFrame = do ->
+setTimeout (->
```

Comment:
@gdh1995.  Thanks for this.

Effectively, you're running this on "nextTick".  How do we know that `createElement` will work then, if it won't work now?  I kinda expected to see `DomUtils.documentReady` here.

(Separately -- and for future reference -- we have `Utils.setTimeout()` and `Utils.nextTick()`.)